example-conf/mesos-dns/mesos-site.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+  <property>
+    <name>mesos.hdfs.mesosdns</name>
+    <value>true</value>
+    <description>If you want mesos dns to be used at all.</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.mesosdns.domain</name>
+    <value>mesos</value>
+    <description>The domain is usually 'mesos', but if you changed this in Mesos DNS, change it here.</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.state.zk</name>
+    <value>master.mesos:2181</value>
+    <description>HDFS framework's path for storing state information in a zookeeper quorum. In this example, we use the same quorum as mesos uses for leader election.</description>
+  </property>
+
+  <property>
+    <name>mesos.master.uri</name>
+    <value>zk://master.mesos:2181/mesos</value>
+    <description>The zookeeper state information to detect the leading mesos master.</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
+    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
+    <description>This is where you must individually list all your zookeeper znodes for Hadoop's HA feature. If you colocate mesos master and zookeeper znodes, you can use Mesos DNS for this.</description>
```

Comment:
These are not znodes, but zk hosts/ports. A `znode` is a key in the `host:port/znode` namespace for a particular value in zk.